dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -91,16 +91,32 @@ public static void assertHasOnlyErrors( ValidationReport report, ValidationCode.
         assertHasErrors( report, codes.length, codes );
     }
 
+    /**
+     * assertHasWarnings asserts the report contains only given warnings in any
+     * order.
+     *
+     * @param report import report to be asserted on
+     * @param warnings expected warnings
+     */
+    public static void assertHasOnlyWarningsAndNoErrors( ImportReport report, Warning... warnings )
+    {
+        assertAll(
+            () -> assertNoErrors( report ),
+            () -> assertHasWarnings( report.getValidationReport(), warnings.length, warnings ) );
+    }
+
     /**
      * assertHasWarnings asserts the report contains only given warnings in any
      * order.
      *
      * @param report validation report to be asserted on
      * @param warnings expected warnings
      */
-    public static void assertHasOnlyWarnings( ValidationReport report, Warning... warnings )
```

Comment:
I would like for us to keep the symmetry we had in the API with regards to error/warning assertions.

Like

```
assertHasOnlyErrors
assertHasOnlyWarnings
```

So if you want to create a new assertion based on the above like `assertHasOnlyWarningsAndNoErrors` I would still keep the lower level ones for error/warning.